packages/stylist/src/yew/mod.rs

```diff
@@ -110,7 +109,12 @@ impl From<Style> for Classes {
 impl From<StyleSource<'_>> for Classes {
     fn from(style_src: StyleSource<'_>) -> Self {
         let mut classes = Self::new();
-        classes.push(style_src.to_style().get_class_name().to_string());
+        #[cfg(debug_assertions)]
+        let location = style_src.location.clone();
```

Comment:
Location should be calculated here instead of style source definition?

So even for the same style source, it can still emit the correct location.

(And can also be added to Style as well.)

I didn't test this so not 100% sure if this is possible.

Reply:
If one has a helper function that creates the style, I wanted the location to be at the place where the `StyleSource` was created, not where it is converted to `Classes`. The location here will be the location of the `html! {}` statement but that might not be the place where the `StyleSource` is created. Probably only differs by a few lines and can be traced back by the user anyway, but I think tracking it in `StyleSource` is more accurate.

Reply:
Makes sense to me.